### ISSUE TITLE
Downloading images?

### ISSUE THREAD

**Issue by lexiross** [peeinears/MagicEye.js#15]
I want to use this library to programmatically generate thousands of magic eye images with digits or words on them for the purpose of building a classifier. It looks like I need to be in a browser context for the library to work. Is there a way I can skip the <img> tag and write the images directly to disk via a node script? Happy to work on a PR for this, but I'm not sure where I'd start :) Thanks!

**Comment by peeinears** [comment]
You're right -- it would need a little work to be able to run outside of a browser.

The guts that actually [convert depth data into magic eye pixel data](https://github.com/peeinears/MagicEye.js/blob/master/magiceye.js#L93-L178) are not dependent on any browser stuff, and you should be able to easily pass the resultant pixel data from that into some JS image encoder ([like this one maybe?](https://github.com/niegowski/node-pngjs)).

The more challenging part might be generating the initial depth data without a browser. The code I have that converts [images](https://github.com/peeinears/MagicEye.js/blob/master/depthmappers/ImgDepthMapper.js) and [text](https://github.com/peeinears/MagicEye.js/blob/master/depthmappers/TextDepthMapper.js) into depth data both rely on HTML Canvas. The ["template" depth mapper](https://github.com/peeinears/MagicEye.js/blob/master/depthmappers/TemplateDepthMapper.js) does *not* rely on Canvas however, so maybe that could work.

Hope that's helpful!

**Comment by lexiross** [comment]
@peeinears thanks so much! I will give it some thought - definitely seems doable

**Comment by lexiross** [comment]
Hey @peeinears, I submitted a PR for this a couple weeks ago - #16 - just wanted to bump it and see if you had a chance to take a look! :)
